### DISCUSSION TITLE
Code Search fails to search our private terraform repo

### DISCUSSION THREAD

**Discussion by drop-nicholas** [community/community#13919] [Other Feature Feedback, Questions, & Ideas]
A simple text search is failing in our private terraform repo.
`repo:earnwithdrop/terraform "traffic for"` is the search and it should have hits.

The repo does return some searches but these files are arbitrarily being ignored.

**Comment by colinwm** [comment]
Hey @drop-nicholas, thanks for the feedback. Note that currently, not all repository content is indexed. See our list of exceptions to what gets indexed here: https://cs.github.com/about/faq#indexed-content

Some files get detected by our systems as being vendored or generated code and are excluded from the index. I realize that these rules can be confusing, and they often get the classification wrong - I'd guess that's what's happening here. Can you provide an example of a filename of a file that should match but doesn't?

It's our goal to remove these rules and index all code, but for now that's unfortunately the limitation.

**Reply by drop-nicholas** [reply]
I'm not sure I see that being relevant here

* :question: Vendored and generated code is excluded (as determined by [Enry](https://github.com/go-enry/go-enry)) -- are `.tf` files vendored? It's called `security_groups.tf`
* :heavy_check_mark: Empty files and files over 350 KiB are excluded -- it's 3.8K
* :heavy_check_mark: Only UTF-8 encoded files are included -- all files UTF-8
* :heavy_check_mark: Very large repositories may not be indexed -- it's not large, the .git folder is 5.4M

Is terraform somehow a vendored file?
Otherwise we're missing a clue.

**Reply by colinwm** [reply]
Hmm, is it possible that part of the full file path would cause it to be detected as vendored/generated? E.g. directories like `third_party/` `vendored/`, `node_modules/`, `cache/` and some others can cause things to be marked as vendored...

**Reply by drop-nicholas** [reply]
It doesn't seem it? 
It's `env/prod/us-east-2/eks/`.

**Reply by drop-nicholas** [reply]
Is there any sort of `.github-cs-include` file that could mark items as desirable to scan?
